src/loglevels.cpp

```diff
@@ -19,7 +19,11 @@ namespace g3 {
 
 #ifdef G3_DYNAMIC_LOGGING
       const std::map<int, LoggingLevel> g_log_level_defaults = {
-         {g3::kDebugValue, {DEBUG}},
+#if (defined(CHANGE_G3LOG_DEBUG_TO_DBUG))
+		  { DBUG.value, { DBUG } },
+#else
```

Comment:
It's cramming too much into a single pull request which might mean it's take longer time to get it in but I truly appreciate that you took the time to address this issue

Thanks